### PR TITLE
Implementation of Workflow API

### PR DESCRIPTION
**Pull request by alfnav**
Full implementation of the Bynder Workflow API:

- Workflow campaigns
- Workflow jobs
- Workflow metaproperties
- Workflow users
- Workflow groups

### CODE REVIEW

**Comment by TimBloembergen** [comment]
Thanks @alfnav. We will review this PR and will keep you posted on what we will do with it.

File: src/main/java/com/bynder/sdk/service/BynderClient.java [hunk 1]
```diff
@@ -52,6 +53,8 @@ public interface BynderClient {
      */
     Observable<Response<List<Derivative>>> getDerivatives();
 
+    WorkflowService getWorkflowService();
```
Comment (Arpit-Sharma-USC) [inline src/main/java/com/bynder/sdk/service/BynderClient.java:56]:
```suggestion
     /**
     * Gets an instance of the workflow service to interact with the workflows in your Bynder
     * portal.
     *
     * @return Instance of {@link WorkflowService}.
     */
    WorkflowService getWorkflowService();
```

File: src/main/java/com/bynder/sdk/service/BynderClientImpl.java [hunk 2]
```diff
@@ -53,6 +54,8 @@ public class BynderClientImpl implements BynderClient {
      */
     private CollectionService collectionService;
 
+    private WorkflowService workflowService;
```
Comment (Arpit-Sharma-USC) [inline src/main/java/com/bynder/sdk/service/BynderClientImpl.java:57]:
```suggestion
    /**
     * Instance of {@link WorkflowService}.
     */
    private WorkflowService workflowService;
```

File: src/main/java/com/bynder/sdk/service/workflow/WorkflowService.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    class Builder {
+
+        private Builder() {
+        }
+
+        public static WorkflowService create(final BynderApi bynderApi, final QueryDecoder queryDecoder) {
+            return new WorkflowServiceImpl(bynderApi, queryDecoder);
+        }
+    }
+}
```
Comment (Arpit-Sharma-USC) [inline src/main/java/com/bynder/sdk/service/workflow/WorkflowService.java:74]: Can you please reformat this file? The indentation is a bit off.

**Review by Arpit-Sharma-USC — COMMENTED** [review]
great work @alfnav! Just left some formatting and documentation suggestions for you.

**Comment by TimBloembergen** [comment]
Hi @alfnav could you please review @Arpit-Sharma-USC 's comments and add the changes requested? We would like to merge these new features and make a new release this week. Thanks!

**Review by Arpit-Sharma-USC — APPROVED** [review]

**Comment by alfnav** [comment]
Hello @TimBloembergen , I am so sorry for the delay in come back to you, I was quite busy at work and i could not find time to do the required changed. Meanwhile I can see that @Arpit-Sharma-USC has already done some of the formatting and documentation changes, sorry for not including this at the beginning. I think that there are also some extra documentation which can be added in the bynder API class and I can do that at a later stage together with other changes in the SDK which I will push if this is ok for you. Thanks!

**Comment by TimBloembergen** [comment]
Hi @alfnav No worries at all, I indeed asked if Arpit could do the changes so we're not blocked for a release. We really appreciate the work and effort you put into this. I believe this is the biggest contribution someone has ever done on any of our SDK's. We will create a new release, if you want to add more documentation to it we would appreciate it so we can merge that at a later stage.

**Review by betacar — APPROVED** [review]
Leaving aside the outdated copyright headers, LGTM! 🐑 🇮🇹

**Pull request merged**